website/docs/components/badge/partials/guidelines/guidelines.md

```diff
@@ -11,112 +27,110 @@ There are six colors: `neutral`, `neutral-dark-mode`, `highlight`, `critical`, `
+### Alternative color usage
 
-!!! Info
+Badges can also indicate different account levels: 
 
-An alternative use case for **Critical Filled**, **Neutral Filled**, and **Warning Filled** are for Account Levels.
+<Hds::Badge @color="critical" @type="filled" @icon="award" @text="Bronze" />
+<Hds::Badge @color="neutral" @type="filled" @icon="award" @text="Silver" />
+<Hds::Badge @color="warning" @type="filled" @icon="award" @text="Gold" />
 
-!!!
+## Type
 
-### Icon
-
-An icon can be shown in addition to or instead of text.
-
-<Hds::Badge @color="neutral" @type="filled" @text="Text only" />
-<Hds::Badge @color="neutral" @type="filled" @icon="corner-down-left" @isIconOnly={{true}} @text="" />
-<Hds::Badge @color="neutral" @type="filled" @icon="hexagon" @text="Text + Icon" />
+There are three types of Badges: `filled`, `inverted`, and `outlined`.
 
-Badges come in a few icon and text combinations; text only, icon only, and icon + text (where icons are always on the left). Use icons intentionally and only when they provide the user with extra value.
+<Hds::Badge @text="Filled" @type="filled" />
+<Hds::Badge @text="Inverted" @type="inverted" />
+<Hds::Badge @text="Outlined" @type="outlined" />
 
-!!! Info
+### Filled
 
-We realize that the small badges use an icon size that is not supported by our icons and thus have scaled it down to fit within the badge. We weighed the pros and cons of doing this versus using the 16px size in the smaller badges and ultimately landed here as it works better visually. You should still be able to use the 16px size within the badge and not see any issues, but please let us know if you do.
+<Hds::Badge @text="Neutral filled" />
+<Hds::Badge @color="highlight" @text="Highlight filled" />
+<Hds::Badge @color="success" @text="Success filled" />
+<Hds::Badge @color="warning" @text="Warning filled" />
+<Hds::Badge @color="critical" @text="Critical filled" />
 
-!!!
+Use filled Badges when displaying many on a single page or for subtle callouts.
 
----
+For example:
 
-### Size
+- when listing statuses in a table
+- for successful or passive actions
 
-There are three sizes: `small`, `medium`, and `large`.
+### Inverted 
 
-<Hds::Badge @text="Small" @size="small" />
-<Hds::Badge @text="Medium" @size="medium" />
-<Hds::Badge @text="Large" @size="large" />
+<Hds::Badge @type="inverted" @text="Neutral inverted" />
+<Hds::Badge @type="inverted" @color="highlight" @text="Highlight inverted" />
+<Hds::Badge @type="inverted" @color="success" @text="Success inverted" />
+<Hds::Badge @type="inverted" @color="warning" @text="Warning inverted" />
+<Hds::Badge @type="inverted" @color="critical" @text="Critical inverted" />
 
-*Medium** is the preferred size, but use a badge size that best fits the UI.
+Use inverted Badges when needing to draw extra attention to something. However, avoid using too many inverted Badges on one page, as it could overwhelm the user and reduce the importance of nearby elements.
```

Comment:
```suggestion
Use inverted Badges to draw extra attention to something. However, avoid using too many inverted Badges on one page, as it could overwhelm the user and reduce the importance of nearby elements.
```